flow-lit-template/src/main/java/com/vaadin/flow/component/littemplate/internal/LitTemplateParserImpl.java

```diff
@@ -107,13 +111,14 @@ public TemplateData getTemplateContent(Class<? extends LitTemplate> clazz,
             }
 
             String url = dependency.getUrl();
-            String source = getSourcesFromTemplate(tag, url);
-            if (source == null) {
-                try {
+            String source;
+            try {
+                source = getSourcesFromTemplate(service, tag, url);
+                if (source == null) {
                     source = getSourcesFromStats(service, url);
```

Comment:
```suggestion
                if (source == null) {
                  if(FeatureFlags.get(service.getContext()).isEnabled(FeatureFlags.VITE)) {
                    source = getTemplateFromServer(url);
                  } else {
                    source = getSourcesFromStats(service, url);
```
Vite shouldn't try get from stats.

Reply:
Getting sources from stats now removed for both Webpack and Lit.